lib/mbta_metro/live/map.ex

```diff
@@ -10,6 +10,7 @@ defmodule MbtaMetro.Live.Map do
     * `:lines` - A list of lists of coordinates to draw lines on the map. Each list of coordinates should be a list of two numbers: the longitude and latitude.
     * `:pins` - A list of coordinates to place pins on the map. Each coordinate should be a list of two numbers: the longitude and latitude.
     * `:points` - A list of coordinates to place points on the map. Each coordinate should be a list of two numbers: the longitude and latitude.
+    * `:icons` - A list of maps with `:coordinates`, `:name`, and `:type` keys to place custom icon markers on the map. Each coordinate should be a list of two numbers: the longitude and latitude. The `:name` and `:type` correspond to the icon component's attributes.
```

Comment:
Could be nice to add `:class` to this to enable custom overrides, but that can be done later.